tools/api-markdown-documenter/src/test/snapshots/simple-suite-test/default-config/simple-suite-test.md

```diff
@@ -263,10 +265,9 @@ An inline type
 
 ### testFunctionReturningIntersectionType {#testfunctionreturningintersectiontype-function}
 
-> **\[Warning\]: Deprecated**
-> 
-> This is a test deprecation notice. Here is a [link](./simple-suite-test#testfunctionreturninguniontype-function) to something else!
-> 
+**WARNING: This API is deprecated and will be removed in a future release.**
+
+_This is a test deprecation notice. Here is a_ [_link_](./simple-suite-test#testfunctionreturninguniontype-function)<!-- --> _to something else!_
 
 Test function that returns an inline type
```

Comment:
Question for reviewers: would it make more sense to move the summary content _above_ the deprecation and alpha/beta alerts in the docs? The default syntax doesn't offer quite as nice a separation between the alerts and the summary, so having the summary first might make more sense?

This is still configurable for consumers by modifying the transformation config, so this would only affect the default behavior.

Reply:
I went ahead and made the update, though if folks have differing opinions, I am still interested in hearing them :)

Reply:
I'm kinda split. It seems like it would be more useful above the summary, especially for particularly long summaries, but having it under the summary matches the syntax better.

Reply:
Yeah. If Markdown supported something like alerts more explicitly, I would definitely prefer above too. As is, I'm not sure.

The lack of delimiter between 2 paragraphs of content is a bummer, and summary above better lines up with how the docs are generally written in the code.

One alternative option would be to add a "Summary" header between the root header and the summary (right now we just put it as a plain paragraph under the root heading). Then we could put the warnings above "Summary".

I can also just put things back for now if you (or anyone else) feels strongly about warnings going above the summary by default.

Reply:
The plan is to restore the "AlertNode" behavior and layout in the ff.com build to preserve existing behavior, so this won't have an impact on our website regardless.

Reply:
Yeah, I think it's fine.